packages/dataviews/src/types.ts

```diff
@@ -497,3 +499,10 @@ export interface SupportedLayouts {
 	grid?: Omit< ViewGrid, 'type' >;
 	table?: Omit< ViewTable, 'type' >;
 }
+
+export interface DataFormProps< Item > {
```

Comment:
This goes against what I said before about Props being collocated. That said, this one is being used in three places. Maybe we can add the `DataViewsProps` here as well.